scipy/linalg/meson.build

```diff
@@ -113,6 +113,22 @@ py3.extension_module('_decomp_interpolative',
   subdir: 'scipy/linalg'
 )
 
+
+py3.extension_module('_batched_linalg',
```

Comment:
If you don't do this as a static (and shared for BLAS/LAPACK dependency) library, other C code can't use it. That's why it is just a header file in common utils right now for sqrtm and eventually for others.